cmd/podman/cleanup.go

```diff
@@ -55,12 +56,25 @@ func cleanupCmd(c *cliconfig.CleanupValues) error {
 	ctx := getContext()
 
 	for _, ctr := range cleanupContainers {
-		if err = ctr.Cleanup(ctx); err != nil {
-			if lastError != nil {
-				fmt.Fprintln(os.Stderr, lastError)
+		hadError := false
```

Comment:
an't we just check if lastError == nil?  Rather then adding hadError?

Reply:
If we're the second container and the first one failed, lastError is always set, even if we did not fail - we need something that's set for each run of the loop and cleared between